### ISSUE TITLE
Custom runtime with MLServer

### ISSUE THREAD

**Issue by oha95** [kserve/modelmesh-serving#222]
I tried to follow the [documentation](https://github.com/kserve/modelmesh-serving/blob/main/docs/runtimes/mlserver_custom.md) to create a custom runtime server 

but I think there is an error in the Dockerfile. 
Is this the last instruction :
```
RUN mlserver start $MLSERVER_MODELS_DIR
```
in that case the image never finish the build. It is not rather: 
```
CMD mlserver start $MLSERVER_MODELS_DIR
```

**Comment by njhill** [comment]
Thanks @oha95 it does look like this is a doc bug. Would you like to open a PR to fix it?

**Comment by rafvasq** [comment]
This doc bug was addressed by #277
